.travis.yml

```diff
@@ -27,16 +27,22 @@ before_install:
 
 matrix:
   fast_finish: true
-  # Allow nightly to fail
   allow_failures:
     - rust: nightly
-  # Only check formatting when running against stable
   include:
+    # Some env var is always necessary to differentiate included builds
+    # Check coding style
     - env: CHECK_FORMATTING=true
+      rust: 1.29.2
```

Comment:
Could we make this 'stable', so we're always using the current stable formatting rules?

Reply:
Using `stable` is what I was trying to avoid, because it tends to break our builds as soon as a new stable is released. Pinning this allows us to upgrade our Rust in our own pace.

Reply:
That's why I made it `stable` initially — so we'd be incentivized to stay on the latest stable.